### PR TITLE
Statistics section and safety plot example added

### PR DESCRIPTION
**Pull request by ClaraBeckBayer**
- Section for statistics in R with link to Phuse working group was added
- FDA safety guidance was added as pdf to Matt scatterplot section because link was not working anymore 
- New safety plot from this document was added in safety plot section, also link to falcon package where all implementations of this guide will live

### CODE REVIEW

File: 06-plots-basics.Rmd
Comment (mattkumar) [inline 06-plots-basics.Rmd:1]: can you change this to eval = T

File: 11c-plots-examples-safety.Rmd
Comment (mattkumar) [inline 11c-plots-examples-safety.Rmd:1]: Can you add text / description for each pivotal step?

File: 11c-plots-examples-safety.Rmd [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+```
+
+```{r, message = FALSE, warning = FALSE, error = FALSE}
+
+# create plot
+
+p <- data %>%
+  filter(!AVISIT %in% c("Baseline", "End of Treatment") & !is.na(AVISIT)) %>%
+  group_by(AVISIT, TRTA) %>%
+  mutate(mean_chg = mean(CHG, na.rm = TRUE),
```
Comment (mattkumar) [inline 11c-plots-examples-safety.Rmd:46]: do we have a source for the formulas/methods? we should ensure correctness

File: 11c-plots-examples-safety.Rmd
Comment (ClaraBeckBayer) [reply 11c-plots-examples-safety.Rmd:1]: you mean markdown text or within the code?

File: 11c-plots-examples-safety.Rmd (hunk 1 above)
Comment (ClaraBeckBayer) [reply 11c-plots-examples-safety.Rmd:46]: No, I created this based on what I saw on the FDA guide plots. However, I'd expect that the specific statistical requirements would be defined by the study and have to be adjusted by the end user when they copy the code. The idea of this script here is more to give an idea of how you could approach a plot like this.

File: 11c-plots-examples-safety.Rmd
Comment (mattkumar) [reply 11c-plots-examples-safety.Rmd:1]: markdown text - kinda describe the code steps of what you're doing/why/any assumptions

Comment (ClaraBeckBayer) [reply 11c-plots-examples-safety.Rmd:1]: Done :-)

File: 11c-plots-examples-safety.Rmd (hunk 1 above)
Comment (ClaraBeckBayer) [reply 11c-plots-examples-safety.Rmd:46]: Do you agree with leaving it as it is?

Comment (mattkumar) [reply 11c-plots-examples-safety.Rmd:46]: The markdown text you added helps, particularly for this step!

**Review by mattkumar — APPROVED** [review]
LG2M

**Pull request merged**